it/test/helpers/agent/ComponentSpecBase.scala

```diff
@@ -93,6 +93,7 @@ trait ComponentSpecBase extends TestSuite with CustomMatchers
     "microservice.services.contact-frontend.port" -> mockPort,
     "feature-switches.read-from-mongo" -> "false",
     "feature-switch.enable-time-machine" -> "false",
+    "feature-switch.enable-user-session-API" -> "true",
```

Comment:
if you can fix naming here as well: 
_feature-switch.enable-session-data-storage_

Reply:
yep thanks